config/example/config.toml

```
@@ -380,7 +380,8 @@ bomb_cap = 20
 revival_brain_life = 6000 # 10 minutes
 # Enable random silicon lawset (If said lawset has default = TRUE in the code). Disable for always crewsimov
 random_ai_lawset = true
-
+# Enable weather events initialized by SSweather. New weather events can still be added during the round if this is disabled.
+enable_default_weather_events = true
 
 ################################################################
```

```suggestion
enable_default_weather_events = true


################################################################
```

Double newline on separators (yes I am this picky)